app/components/characters/ProfileCard/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+
+import Student from '../ts/interfaces/Student'
+import * as Social from '../social-media-icons'
+import * as utils from '../utils/helpers'
+
+import { LowerNotch, UpperNotch, GradientImageDiv } from './styles'
+import { useResponsive } from './hooks'
+import { ProfilePictureContainer } from './ProfilePictureContainer'
+
+const ProfileCard = ({ name, major, image, house }: Student) => {
+	const { isMobile } = useResponsive()
+
+	if (isMobile) {
+		return (
+			<div className='relative'>
+				<ProfilePictureContainer image={image} />
+				<div className='absolute top-1.5 right-1.5'>
+					{utils.getHouseIcon(house)}
+				</div>
+				<GradientImageDiv className='absolute bottom-0 left-0 w-full pt-12 pb-3 px-2'>
+					<p className='text-1.5xs text-white font-bold'>
+						{utils.shortenName(name)}
+					</p>
+					<p className='text-2xs text-white font-extralight'>{major}</p>
+				</GradientImageDiv>
+			</div>
+		)
```

Comment:
can you make it aa a different separated component and render it conditionally on its parent?

Reply:
Could you give me an example of what you mean by this? I still don't quite get it.

Reply:
I mean, could you make it as a separate file. e.g. `ProfileCard/index.tsx` and `ProfileCard/mobilProfileCard.tsx`. and then conditionally render it at `/pages/characters/index.tsx `

Reply:
Oh all right. I'll try that.